addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -733,6 +735,14 @@ registry.mediaVideo = publicWidget.Widget.extend(MobileYoutubeAutoplayMixin, {
             iframeEl = this._generateIframe();
         }
 
+        if (this.el.dataset.needCookiesApproval) {
+            const sizeContainerEl = this.el.querySelector(":scope > .media_iframe_video_size");
+            sizeContainerEl.classList.add("d-none");
+            document.addEventListener("optionalCookiesAccepted", () => {
```

Comment:
Doesn't the fact that the event is only listened once (and what it does is precisely removing the class) already care for what would happen if I did that in `destroy`? :thinking:

Reply:
@robinlej It may be listened only one time **maximum** but would not it be listened at all only if the user does accept optional cookies? If he never clicks on the cookie banner then enter edit mode, is not the listener still active for no reason?